documentation/API-design-guidelines.md

```diff
@@ -1747,7 +1747,7 @@ Remark: This action will trigger a subscription-ends event with terminationReaso
 
 
 **Note** on combined usage of initialEvent and subscriptionMaxEvents: 
-Unless explicitly decided otherwise by the API WG,  if an event is triggered following initialEvent set to `true`, this event will be counted towards subscriptionMaxEvents (if provided).
+Unless explicitly decided otherwise by the API Sub Project,  if an event is triggered following initialEvent set to `true`, this event will be counted towards subscriptionMaxEvents (if provided).
```

Comment:
```suggestion
Unless explicitly decided otherwise by the API Sub Project, if an event is triggered following initialEvent set to `true`, this event will be counted towards subscriptionMaxEvents (if provided).
```

Reply:
fix additional space